Vizzuality/gfw
# Feature/update supported browsers

## Overview

Remove IE11 from the browser utils checks and updates sentence displayed to user when visiting from unsupported browser. Also removes redundant layout.

## Testing

- Set user agent to internet explorer in dev tools
- Visit home page and check you are redirected to `/browser-support/`
